diff -cN a/idea64.vmoptions b/idea64.vmoptions
*** a/idea64.vmoptions	2014-05-10 07:02:16.208497068 +0000
--- b/idea64.vmoptions	2014-05-10 07:00:40.848497759 +0000
***************
*** 1,12 ****
! -Xms128m
! -Xmx750m
  -XX:MaxPermSize=350m
! -XX:ReservedCodeCacheSize=96m
  -ea
  -Dsun.io.useCanonCaches=false
  -Djava.net.preferIPv4Stack=true
  -Djsse.enableSNIExtension=false
  -XX:+UseCodeCacheFlushing
  -XX:+UseConcMarkSweepGC
  -XX:SoftRefLRUPolicyMSPerMB=50
! -Dawt.useSystemAAFontSettings=lcd
--- 1,16 ----
! -Xms1024m
! -Xmx2048m
  -XX:MaxPermSize=350m
! -XX:ReservedCodeCacheSize=256m
! -XX:+UseCodeCacheFlushing
  -ea
+ -server
  -Dsun.io.useCanonCaches=false
  -Djava.net.preferIPv4Stack=true
  -Djsse.enableSNIExtension=false
  -XX:+UseCodeCacheFlushing
  -XX:+UseConcMarkSweepGC
  -XX:SoftRefLRUPolicyMSPerMB=50
! -Dawt.useSystemAAFontSettings=on
! -XX:+DoEscapeAnalysis
! -XX:+UseCompressedOops
