*** a/idea64.vmoptions	2015-02-25 03:20:05.752091457 +0900
--- b/idea64.vmoptions	2015-02-25 03:23:55.232089796 +0900
***************
*** 1,5 ****
! -Xms128m
! -Xmx750m
  -XX:MaxPermSize=350m
  -XX:ReservedCodeCacheSize=225m
  -ea
--- 1,5 ----
! -Xms1024m
! -Xmx2048m
  -XX:MaxPermSize=350m
  -XX:ReservedCodeCacheSize=225m
  -ea
***************
*** 8,11 ****
  -Djsse.enableSNIExtension=false
  -XX:+UseConcMarkSweepGC
  -XX:SoftRefLRUPolicyMSPerMB=50
! -Dawt.useSystemAAFontSettings=lcd
--- 8,15 ----
  -Djsse.enableSNIExtension=false
  -XX:+UseConcMarkSweepGC
  -XX:SoftRefLRUPolicyMSPerMB=50
! -Dawt.useSystemAAFontSettings=on
! -XX:+UseCodeCacheFlushing
! -server
! -XX:+DoEscapeAnalysis
! -XX:+UseCompressedOops
